packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2009,21 +2009,73 @@ class RepaintBoundary extends OneChildRenderObjectWidget {
 }
 
 class IgnorePointer extends OneChildRenderObjectWidget {
-  IgnorePointer({ Key key, Widget child, this.ignoring: true })
+  IgnorePointer({ Key key, Widget child, this.ignoring: true, this.ignoringSemantics })
     : super(key: key, child: child);
 
   final bool ignoring;
+  final bool ignoringSemantics; // if null, defaults to value of ignoring
 
-  RenderIgnorePointer createRenderObject() => new RenderIgnorePointer(ignoring: ignoring);
+  RenderIgnorePointer createRenderObject() => new RenderIgnorePointer(
+    ignoring: ignoring,
+    ignoringSemantics: ignoringSemantics
+  );
 
   void updateRenderObject(RenderIgnorePointer renderObject, IgnorePointer oldWidget) {
     renderObject.ignoring = ignoring;
+    renderObject.ignoringSemantics = ignoringSemantics;
   }
 }
 
 
 // UTILITY NODES
 
+class Semantics extends OneChildRenderObjectWidget {
+  Semantics({
+    Key key,
+    Widget child,
+    this.container: false,
+    this.checked,
+    this.label
+  }) : super(key: key, child: child) {
+    assert(container != null);
+  }
+
+  final bool container;
+  final bool checked;
+  final String label;
+
+  RenderSemanticAnnotations createRenderObject() => new RenderSemanticAnnotations(
+    container: container,
+    checked: checked,
+    label: label
+  );
+
+  void updateRenderObject(RenderSemanticAnnotations renderObject, Semantics oldWidget) {
+    renderObject.container = container;
+    renderObject.checked = checked;
+    renderObject.label = label;
```

Comment:
Filed https://github.com/flutter/flutter/issues/1381. Not sure it's worth it, though. I would hope the resulting code would be identical, and with the current pattern the code is more consistent.